src/dcap_provider.cpp

```diff
@@ -80,20 +80,25 @@ static char PRIMARY_CERT_URL[] =
     "http://169.254.169.254/metadata/THIM/sgx/certification";
 static std::string primary_cert_url = PRIMARY_CERT_URL;

```

Comment:
Don't we need to conditionally compile the primary URL too?

Reply:
Primary url is for service, which we don't use in Service VM scenario.